src/app/components/Figure/index.jsx

```diff
@@ -2,8 +2,18 @@ import React from 'react';
 import { string } from 'prop-types';
 import styled from 'styled-components';
 import Caption from './Caption';
+import VisuallyHiddenText from '../VisuallyHiddenText';
 import { GEL_SPACING_DBL } from '../../lib/constants/styles';
 
+const copyrightOffscreenText = 'Copyright ';
+
+const renderCopyright = copyright =>
+  copyright !== 'BBC' ? (
+    <VisuallyHiddenText>
+      {copyrightOffscreenText + copyright}
```

Comment:
Minor change request: can you please use es6 templating here; then there also won't need to be a blank space on L8.